src/index.test.ts

```diff
@@ -340,3 +416,55 @@ describe('waitUntil', () => {
     ).rejects.toMatch('fail')
   })
 })
+
+describe('throughputLimiter', () => {
+  test('first call to throttle completes without delay', async () => {
+    expect.assertions(2)
```

Comment:
What is the reason for these `expect.assertions` calls? My understanding is that we don't need them, because the `expect` calls below it are guaranteed to be executed. We would need them if the `expect` calls were happening asynchronously, or maybe in a try/catch where the catch might not happen.

Reply:
These are not needed. Just something I do for this library, in particular, since the tests can be a little tricky at times.

Reply:
My initial thought was that it's better not to have them when we don't need them, because it means we have to remember to update the number whenever we add/remove assertions, which is slightly inconvenient.

But I also see how it's useful to have them as a general rule in this code base, so that we're covered just in case we write a test case and don't realize that it needs the `expect.assertions`.

Reply:
I removed them from these tests.